content/pt-br/event-streaming.md

```diff
@@ -0,0 +1,30 @@
+---
+título: Streaming de Eventos
+estado: Concluído
+categoria: conceito
```

Comment:
Probably this translation breaks down the glossary. Could you please revert it?

"title" instead of "título"
"status" instead of "estado"
"category" instead of "categoria"